recipes/k3d/yum_requirements.txt

```diff
@@ -0,0 +1,3 @@
+mesa-libGL-devel
+xorg-x11-server-Xvfb
```

Comment:
Should this be a runtime dep as well, so it picks up the CDT? https://anaconda.org/search?q=xorg-x11-server-Xvfb

Reply:
You mean the mesa libGL specifically? It's there since it's the only real option for the tests AFAIK. I also know nothing about CDT. So I really cannot say. 
All rendering will happen inside the browser, in a jupyter notebook session.

Reply:
Are libGL and xorg only needed for testing?

Reply:
Yes. The package would also be usable on wayland and the browser's software renderer.